hibernate-core/src/test/java/org/hibernate/orm/test/mapping/enumeratedvalue/CharEnumerateValueTests.java

```diff
@@ -122,7 +121,7 @@ void verifyCheckConstraints2(SessionFactoryScope scope) {
 
 	@AfterEach
 	void dropTestData(SessionFactoryScope scope) {
-		scope.inTransaction( (session) -> session.createMutationQuery( "delete Person" ).executeUpdate() );
+		scope.getSessionFactory().getSchemaManager().truncate();
```

Comment:
I wonder ... what if we add something like

```java
public @interface SessionFactory {
	boolean truncate() default true; 
....
```
And do it as part of the extension rather than in the tests? (then we could skip a lot of these after each calls ...)

Reply:
@marko-bekhta Yeah, that would be great.

Reply:
So I gave it a try (truncate after each, evict cache after each):

```
15901 tests

756 failures
```
🙈 🥲